### ISSUE TITLE
Postgres version check fails < 10.0

### ISSUE THREAD

**Issue by kamicut** [Hecate/Hecate#1]
### Context
At boot, the version checker fails because it doesn't account for patch versions. 

`cargo run` fails with:

```
ERROR: Connection unable obtain postgres version using (READ/WRITE) hecate@localhost:5432/hecate
ERROR: database error
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/libcore/option.rs:378:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

The SQL select statement for version check fails with:
```
ERROR:  invalid input syntax for type double precision: "9.6.14"
```

cc/ @ingalls

**Comment by guidorice** [comment]
I am opening a Draft PR that adds semantic version parsing for the postgresql and postgis database check.

**Comment by ingalls** [comment]
Closed in `Hecate@0.86.2`

**Comment by ingalls** [comment]
Thanks a ton to @guidorice !!
